sdk/loadtestservice/Azure.Developer.LoadTesting/Hero/Program.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using Azure;
+using Azure.Core;
+using Azure.Developer.LoadTesting;
+using Azure.Identity;
+
+namespace Hero
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            // creating TokenCredential
+            // Environment.GetEnvironmentVariable
+            TokenCredentialOptions tokenCredentialOptions = new TokenCredentialOptions();
+            tokenCredentialOptions.AuthorityHost = new Uri("https://login.windows-ppe.net");
+            
+            TokenCredential credential = new ClientSecretCredential(
+                    tenantId: Environment.GetEnvironmentVariable("AZURE_TENANT_ID"),
+                    clientId: Environment.GetEnvironmentVariable("AZURE_CLIENT_ID"),
+                    clientSecret: Environment.GetEnvironmentVariable("AZURE_CLIENT_SECRET"),
+                    options: tokenCredentialOptions
+                );
+
+
+            // creating Loadtesting client
+            LoadTestingClient loadTestingClient = new LoadTestingClient(
+                endpoint: "https://4c58f20e-52a9-4e22-8593-33938c025b6b.northeurope.cnt-dev.loadtesting.azure.com",
+                credential: credential
+            );
+
+
+            // getting subclients
+            LoadTestAdministrationClient loadTestAdministrationClient = loadTestingClient.getLoadTestAdministration();
+            LoadTestRunClient loadTestRunClient = loadTestingClient.getLoadTestRun();
+
+
+            // setting few important variables
+            string testId = "my-dotnet-test-id";
+            string testRunId = "my-dotnet-test-run-id";
+            string fileName = "my-dotnet-file-name.jmx";
+            string subscriptionId = Environment.GetEnvironmentVariable("SUBSCRIPTION_ID");
+            string appComponentConnectionString = "/subscriptions/" + subscriptionId + "/resourceGroups/App-Service-Sample-Demo-rg/providers/Microsoft.Web/sites/App-Service-Sample-Demo";
+
+            // create loadtest
+            try
+            {
+                Response response = loadTestAdministrationClient.CreateOrUpdateTest(testId, RequestContent.Create(
+                   new
+                   {
+                       description = "This is created using SDK",
+                       displayName = "SDK's LoadTest",
+                       loadTestConfig = new
+                       {
+                           engineInstances = 1,
+                           splitAllCSVs = false,
+                       },
+                       secrets = new { },
```

Comment:
do we need to provide some meaningful values for some of these? else do we need to remove them instead of specifying empty objects?

Reply:
As per my information, we need to provide empty if we are not providing secrets